services/catalog/src/simcore_service_catalog/api/rpc/_services.py

```diff
@@ -165,3 +167,32 @@ async def check_for_service(
         service_key=service_key,
         service_version=service_version,
     )
+
+
+@router.expose(reraise_if_error_type=(CatalogForbiddenError,))
+@log_decorator(_logger, level=logging.DEBUG)
+async def batch_get_my_services(
```

Comment:
This works the same, but the server validates it isntead of the client, which is more appropriate.

```suggestion
@router.expose(reraise_if_error_type=(CatalogForbiddenError,))
@log_decorator(_logger, level=logging.DEBUG)
@validate_call(config={"arbitrary_types_allowed": True})
async def batch_get_my_services(
```

Reply:
I would also change all the other calls in the client to use this pattern, server side validation

Reply:
Please also change it in the other calls to make it so that it's the server which validates and not the client. Thanks a lot!